spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+RSpec.configure do |config|
+    config.formatter = :documentation
+end
+
+RSpec.describe VehicleFactory do
+    before(:each) do
+        @factory = VehicleFactory.new
+    end
+
+    describe '#initialize' do
+        it 'can initialize' do
+        expect(@factory).to be_an_instance_of(VehicleFactory)
+        end
+    end
+
+    describe '#create_vehicles' do
+        it "can create an array of vehicles objects" do
+            wa_ev_registrations = DmvDataService.new.wa_ev_registrations
```

Comment:
You can remove this `wa_ev_registrations` and just use the test data you're defining on the next line if you'd like! This is a great example of using fixed test data (even if it originally came from an external source) in order to have control over what the data coming in will be.